rover/source/docs/sailboat-hardware.rst

```diff
@@ -11,7 +11,7 @@ way. I.e. no change of servo position should be required to set the sails at the
 achieved through the use of a sail winch servo with a main sheet system. The sail winch is servo output function, ``SERVOx_FUNCTION``, 89 - Main Sail.
 
 Although not absolutely required for best results a wind vane should be fitted to allow the vehicle to sense the wind 
-direction. See :ref:<wind-vane>. If a wind vane sensor is not used, then assumed wind direction must be input using these methods: :ref:`wind-vane-home-heading`.
+direction. See :ref:`wind vane <wind-vane>`. If a wind vane sensor is not used, then assumed wind direction must be input using these methods: :ref:`wind-vane-home-heading`.
```

Comment:
probably should capitalize the W and V in the first part of the ref....easier is just to eliminate it and use :ref:`wind-vane` since it will display the tags title from the page

Reply:
note the backticks do not appear in the above comment and are interpreted in MD as grey highlighting...hope you can figure it out

Reply:
@Hwurzburg I'm pretty handy with MD, less familiar with RST. I simply coped and pasted a working version of the same link from a couple of lines down. Feel free to change it how you like it, or close this PR entirely. It's a small thing.

This is how the excerpt renders in current master. THis is what I Was trying to fix:

![Screenshot_2021-04-18_12-39-14](https://user-images.githubusercontent.com/1756197/115155015-79b39f80-a043-11eb-9edf-c3e027bcb3e9.png)